<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectsock</class>
 <widget class="QWidget" name="selectsock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>筛选</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/socks_icon_1.png</normaloff>:/socks_icon_1.png</iconset>
  </property>
  <widget class="QPushButton" name="btn_select">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_position">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_position">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>仓储位置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>单价</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_minPrice">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_maxPrice">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_other">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>combo_position</tabstop>
  <tabstop>line_minPrice</tabstop>
  <tabstop>line_maxPrice</tabstop>
  <tabstop>btn_select</tabstop>
  <tabstop>btn_exit</tabstop>
 </tabstops>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
